### ISSUE TITLE
Support Ed25519 SSH keys

### ISSUE THREAD

**Issue by shaneparsons** [cweijan/vscode-database-client#264]
`Ed25519` keys seem un-supported by the plugin:
```
2021-08-02 13:26:38 Ssh tunel occur error : Error: Cannot parse privateKey: Unsupported key format
```
Can you please add support for these kinds of keys? `Ed25519` is being referred to as "the most recommended public-key algorithm", so I expect more and more people to run into this issue.

**Comment by cweijan** [comment]
It's not that I don't want to support it, but that the ssh library does not support it .

**Comment by songlineboy** [comment]
will support for ecdsa / 384 be added? I assume is this: ecdsa-sha2-nistp384

 I have the same issue as about. thanks.

**Comment by cweijan** [comment]
Fix at version 4.2.3.
